proposals/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Solid Data Interoperability Panel: Proposals</title>
+</head>
+
+<body>
+  <ul>
+    <li><a href="problems-and-goals/">Problems and Goals for Interoperability in Solid</a></li>
+    <li><a href="specification/">Solid Application Interoperability</a></li>
+    <li><a href="primer/">Solid Primer</a></li>
```

Comment:
Should be _Solid Application Interoperability Primer_?

Reply:
done